ansible/roles/controller/tasks/main.yml

```diff
@@ -16,6 +16,14 @@
     - python2-requests
     - git
 
+- name: Creates docker directory
+  file: path=/etc/docker/ state=directory
```

Comment:
Please use consistent argument style, like in `Set up docker bridge network` task below.